change/@azure-communication-react-0318bc34-55d1-4d98-a106-ee5dbbf0deca.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix CI issue of \"No files matching the pattern \"*/**/*.tsx\" were found\"",
+  "packageName": "@azure/communication-react",
+  "email": "107075081+Leah-Xia-Microsoft@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

Comment:
Anything does not impact our production code would be considered as "none"

Only those PRs patch the ui library package would be type "patch"